.github/workflows/ci.yml

```diff
@@ -6,7 +6,12 @@ env:
   BUILD_TYPE: Debug
 
 jobs:
+ 
+  ## LINTING JOBS  
+  linting:
+    uses: ./.github/workflows/linting.yml
```

Comment:
Linting is a separate workflow, it does not expect any parameter from the `ci` workflow, nor does it create any artifact for future usage. Maybe it could be an independent workflow, executed on `push`, `pull_request` and `workflow_dispatch`, the same as the `ci` workflow.